react/src/pages/MaintenancePage.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Button, Card, Row, Col } from 'antd';
+import { RedoOutlined } from '@ant-design/icons';
+import { useTranslation, Trans } from "react-i18next";
+import { useWebComponentInfo } from '../components/DefaultProviders';
+
+const MaintenancePage : React.FC = () => {
+	const { t } = useTranslation();
+
+	const { value, dispatchEvent } = useWebComponentInfo();
+	let parsedValue: {
+		recalculating: boolean;
+		scanning: boolean;
+	};
+	try {
+		parsedValue = JSON.parse(value || "");
+	} catch (error) {
+		parsedValue = {
+			recalculating: false,
+			scanning: false
+		};
+	};
+	const { recalculating, scanning } = parsedValue
+
+	return(
+		<Row gutter={14}>
+			<Col>
+				<Card
+					className="matchDatabase"
+					title={t("maintenance.Fix")}
+					bodyStyle={{ paddingTop: "10px" }}
+				>
+					<p
+						className="title"
+						style={{
+							fontSize: "14px",
+							fontWeight: "bold"
+						}}
+					>
+						{t("maintenance.MatchDatabase")}
+					</p>
+					<p
+						className="description"
+						style={{
+							fontSize: "13px",
+							marginTop: "5px",
+							marginRight: "5px"
+						}}
+					>
+						<Trans>
+							{t("maintenance.DescMatchDatabase")}
+						</Trans>
+					</p>
+					<Button
+						block
+						disabled={recalculating}
+						icon={<RedoOutlined />}
+						style={{
+							color: "#38bd73",
+							borderWidth: "2px",
+							borderColor: "#e0e0e0",
+							fontWeight: 400,
+							padding: "15px",
+							justifyContent: "center",
+							display: "flex"
+						}}
+						onClick={()=> dispatchEvent("recalculate", null)}
+					>
+						{recalculating ? t("maintenance.Recalculating") : t("maintenance.RecalculateUsage")}
+					</Button>
```

Comment:
ou don't need to make the React-based component look exactly the same as the previous component. The React-based component on the WebUI adopts the Ant Design System, so most of the time, you don't need to explicitly set font weight, color, size, etc.

You can change this part like this:
```jsx
<Card title={t("maintenance.Fix")}>
	<Typography.Title level={5} style={{ margin: 0 }}>
		{t("maintenance.MatchDatabase")}
	</Typography.Title>
	<Trans>{t("maintenance.DescMatchDatabase")}</Trans>
	<Button
		block
		disabled={recalculating}
		icon={<RedoOutlined />}
		onClick={() => dispatchEvent("recalculate", null)}
	>
		{recalculating
			? t("maintenance.Recalculating")
			: t("maintenance.RecalculateUsage")}
	</Button>
</Card>
```

Reply:
Thank you. I will request review again after modiying according your comment.😊

Reply:
@yomybaby
I modified code accroding your review! If you have time, please review one more time